app.py

```diff
@@ -155,8 +163,13 @@ def initapp():
 	if app.config['ACCOUNTS'] is None:
 		raise ValueError('ACCOUNTS not configured')
 	# read sentences to annotate
-	with open(sentfilename) as sentfile:
-		SENTENCES = sentfile.read().splitlines()
+	sentences = []
+	with open(sentfilename, mode='r') as csv_file:
+		csv_reader = csv.DictReader(csv_file)
```

Comment:
The CSV format means sentences containing commas would have to be quoted, and quotes in the sentence would have to be escaped, right? What about using a tab-separated format instead?

Reply:
I think `csv.DictReader(csv_file, delimiter='\t', quoting=csv.QUOTE_NONE)` would achieve this